tests/performance_tests/performance_test_basic.py

```diff
@@ -294,17 +297,30 @@ def readUserTrxDataFromFile(self, userTrxDataFile: Path):
             self.userTrxDataDict = json.load(f)
 
     def setupContract(self):
-        specifiedAccount = Account(self.clusterConfig.specifiedContract.accountName)
-        specifiedAccount.ownerPublicKey = self.clusterConfig.specifiedContract.ownerPublicKey
-        specifiedAccount.activePublicKey = self.clusterConfig.specifiedContract.activePublicKey
-        self.cluster.createAccountAndVerify(specifiedAccount, self.cluster.eosioAccount, validationNodeIndex=self.validationNodeId)
-        print("Publishing contract")
-        transaction=self.cluster.biosNode.publishContract(specifiedAccount, self.clusterConfig.specifiedContract.contractDir,
-                                                          self.clusterConfig.specifiedContract.wasmFile,
-                                                          self.clusterConfig.specifiedContract.abiFile, waitForTransBlock=True)
-        if transaction is None:
-            print("ERROR: Failed to publish contract.")
-            return None
+        if (self.clusterConfig.specifiedContract.accountName != self.cluster.eosioAccount.name):
+            specifiedAccount = Account(self.clusterConfig.specifiedContract.accountName)
+            specifiedAccount.ownerPublicKey = self.clusterConfig.specifiedContract.ownerPublicKey
+            specifiedAccount.ownerPrivateKey = self.clusterConfig.specifiedContract.ownerPrivateKey
+            specifiedAccount.activePublicKey = self.clusterConfig.specifiedContract.activePublicKey
+            specifiedAccount.activePrivateKey = self.clusterConfig.specifiedContract.activePrivateKey
```

Comment:
The code here matches the format of the code below and can be put into a function

Reply:
@ClaytonCalabrese What were your thoughts here?  It is indeed setting the 4 keys in each scenario, however, it is on two different object types so I'm not sure it condenses nicely into a simple, easily reusable function.  Were you thinking a function that takes either the `Account` and/or `SpecifiedContract` and sets the 4 keys on either one of them based on which is provided? Or a helper in `Account` and in `SpecifiedContract` that takes 4 keys and sets them on `self`?

Reply:
After thinking about this more, what I said doesn't make sense and I misunderstood what was happening. This code will get resolved at some point anyways once our two PRs meet so any changes here would probably end up pointless anyways.